gateway/cli/src/general_commands.rs

```diff
@@ -56,30 +50,8 @@ pub enum GeneralCommands {
         #[clap(long)]
         federation_id: Option<FederationId>,
     },
-    /// Check gateway's e-cash balance on the specified federation.
```

Comment:
We might consider leaving the old commands for time being to help with the transition.
We don't have to do it. But we might consider it. :D

Reply:
With the `gateway-cli` we don't try to maintain backwards compatibility, so I would prefer just to move them :)

Reply:
They're not! Which is nice, we added some logic in the backwards compatibility tests that always ensure `gateway-cli` version and `gatewayd` version are the same.

Reply:
Go for it then. :)